pkg/registry/apis/provisioning/register.go

```diff
@@ -814,3 +822,58 @@ func (b *APIBuilder) encryptSecrets(ctx context.Context, repo *provisioning.Repo
 	}
 	return nil
 }
+
+// When starting an empty instance -- switch to mode 5
+func (b *APIBuilder) tryRunningOnlyUnifiedStorage() error {
+	ctx := context.Background()
+	if !dualwrite.IsReadingLegacyDashboardsAndFolders(ctx, b.storageStatus) {
+		return nil
+	}
+
+	// Count how many things exist
+	rsp, err := b.legacyMigrator.Migrate(ctx, legacy.MigrateOptions{
+		Namespace: "default", // FIXME!!
```

Comment:
default will work for us now -- but this should be updated so it would look at *any* namespace (eg, orgID)

Reply:
let's add extend the FIXME note :)